spec/services/catalog/controllers/admin-catalog-item-controller-spec.js

```diff
@@ -100,13 +96,13 @@ describe('Catalog Service Controller', function () {
 			identity: {
 				platform: {id: 'apple-ios'},
 				user: {id: 'user-id'},
-				audience: 'admin'
+				audience: ['admin']
```

Comment:
While this is technically correct, a string here gets transformed into an array in [request-authorize.js](https://github.com/oddnetworks/oddworks/blob/master/lib/middleware/request-authorize.js#L38).